TWLight/applications/templates/applications/application_list.html

```diff
@@ -147,12 +147,12 @@ <h1>{{ title }}</h1>
               {{ object_list.number }}
             </a>
           </li>
-          {% comment %}Display next page.{% endcomment %}
+          {% comment %} Display next page. {% endcomment %}
           {% if object_list.has_next %}
             <li><a href="?page={{ object_list.next_page_number }}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
                 {{ object_list.next_page_number }}
                 </a></li>
-            {% comment %}Decide based on the total number of pages, if a '...' is needed or not.{% endcomment %}
+            {% comment %} Decide based on the total number of pages, if a '...' is needed or not. {% endcomment %}
```

Comment:
The only thing you're doing here is adding padding around the comment string. In some places, you've made changes and added padding, and in others you haven't. Which should it be?

Reply:
I've stripped the padding at the beginning and end of all `{% comment %}...{% endcomment %}` across the project.